test/metabase/api/dashboard_test.clj

```diff
@@ -203,7 +203,6 @@
                                                            :dataset_query          {}
                                                            :read_permissions       nil
                                                            :visualization_settings {}
-                                                           :query_average_duration nil
```

Comment:
just to ask a dumb question, why was this expected to be nil? 

Is it that the same query is being run while this test executes and the test expects there to have been no runs of the cards in the test dashboard?

Reply:
This test does not run queries. If it were the only test that ran, `query-average-duration` would be `nil`, and it would be correct. But since `query-average-duration` is based on the hash of the query, depending on whether other tests run the same query beforehand, this value might end up being non-nil.

Reply:
👍